charts/irs-helm/values.yaml

```
@@ -203,7 +203,7 @@ edc:
         operator: "eq"
         rightOperand: "active"
   connectorEndpointService:
-    cacheTTL: 86400000
+    cacheTTL: PT24H # Time to live for ConnectorEndpointService for fetchConnectorEndpoints method cache
```

Comments need to be separated by two spaces
```suggestion
    cacheTTL: PT24H  # Time to live for ConnectorEndpointService for fetchConnectorEndpoints method cache
```